dvc/command/repro.py

```diff
@@ -30,6 +30,11 @@ def run(self):
 
                 if len(stages) == 0:
                     logger.info(CmdDataStatus.UP_TO_DATE_MSG)
+                else:
+                    logger.info(
+                        "Use `dvc push` to send your updates to "
+                        "remote storage."
```

Comment:
Minor: Should we update some of the console examples in https://dvc.org/doc/command-reference/repro? Several show the command's output.

> p.s. looks like that could fit in a single-line-string?

Reply:
This is a good point, but there is a problem:
In the example, in `repro` we modify code only, without influence on the output of the stage...
Seems to me like we should not display this help message there. So I would say we should not update the command reference but fix the underlying issue.